homeassistant/components/google_assistant/button.py

```diff
@@ -34,14 +34,18 @@ async def async_setup_entry(
 class SyncButton(ButtonEntity):
     """Representation of a synchronization button."""
 
+    _attr_has_entity_name = True
```

Comment:
Since there is no device name, this is prefixing the device with the project ID, which is not user friendly.